guides/release-notes.md

```diff
@@ -7,6 +7,14 @@ We love hearing your feedback! You can help us improve the docs by reporting any
 ![Was this helpful, yes/no buttons](../media/feedback.png)
 
 ---
+### Deprecation Notice: Workflows API
+
+`wix-workflows.v2`
+
+The [Workflows API](https://www.wix.com/velo/reference/wix-workflows-v2) is being discontinued and will no longer be supported moving forward. As this may impact your existing code, we highly encourage you to explore alternative solutions available within Wix to ensure seamless functionality for your projects. 
```

Comment:
> As this may impact your existing code, we highly encourage you to explore alternative solutions available within Wix to ensure seamless functionality for your projects.

I don't know that this statement is necessary as it really doesn't impact users, according to Itzik and Avraham.

There's no real seamless path forward. I think it's okay to be gently firm here and say that they should remove any code that uses workflows.v2.

I suppose if you want to soften the blow, you can say that crm-backend.v2 will be available until the product is discontinued in June?

Reply:
But `crm-backend.v2` does not help. If you must we can point to `workflows` namespace in `wix-crm-backend`